js/src/main/scala/org/soundsofscala/transport/NoteScheduler.scala

```diff
@@ -45,7 +45,8 @@ case class NoteScheduler(
           case atomicEvent: AtomicMusicalEvent =>
             scheduleAtomicEvent(atomicEvent, nextNoteTime, instrument)
       case ScheduleState.Waiting =>
-        IO.sleep(lookAheadMs.value.millis) >> scheduler(musicalEvent, nextNoteTime, instrument)
+        Temporal[IO]
+          .sleep(lookAheadMs.value.millis) >> scheduler(musicalEvent, nextNoteTime, instrument)
```

Comment:
**suggestion:** when using IO directly, I'd use `IO.sleep` instead of going through a typeclass

Reply:
Ah yes gotcha